drivers/mysql/mysql.go

```diff
@@ -172,14 +172,34 @@ func (driver *mysql) Apply(migration *models.Migration, saveVersion bool) (err e
 	}
 
 	updateVersionQuery := driver.addMigrationQuery(migration)
-	if _, err := driver.conn.ExecContext(updateVersionContext, updateVersionQuery); err != nil {
+	if res, err := driver.conn.ExecContext(updateVersionContext, updateVersionQuery); err != nil {
 		return &drivers.DatabaseError{
 			OrigErr: err,
 			Driver:  driverName,
 			Message: "failed when updating migrations table with the new version",
 			Command: "update_version",
 			Query:   []byte(updateVersionQuery),
 		}
+	} else if res != nil {
```

Comment:
Can there be a case where err is `nil` and res is also `nil`? If not, I think we can just outdent this since we are returning if err != nil.

Reply:
Right, it shouldn't.